docs/source/common/weights_loading.rst

```diff
@@ -216,3 +179,175 @@ do the following:
 
    # automatically restores model, epoch, step, LR schedulers, apex, etc...
    trainer.fit(model, ckpt_path="some/path/to/my_checkpoint.ckpt")
+
+
+-----------
+
+
+*******************************************
+Conditional Checkpointing (ModelCheckpoint)
+*******************************************
+
+The :class:`~pytorch_lightning.callbacks.ModelCheckpoint` callback allows you to configure how & when checkpointing should happen. It follows the normal Callback hook structure so you can
```

Comment:
how & when --> when & which ?

Reply:
how about?
```
The :class:`~pytorch_lightning.callbacks.ModelCheckpoint` callback allows you to configure how, when, and which checkpoints should be saved to storage.
```

Reply:
hm i guess "how" could encompass a lot, though probably avoid "how to storage" as that's very specifically IOPlugin